src/main/java/jenkins/plugins/office365connector/Webhook.java

```diff
@@ -106,22 +104,101 @@ public boolean isNotifyRepeatedFailure() {
     }
 
     public int getTimeout() {
-            return timeout;
+        return timeout == 0 ? DEFAULT_TIMEOUT : timeout;
```

Comment:
0 or -1? Could 0 be valid number for timeout?

Reply:
Java defaults int to zero and wouldn't consider 0 ms as a proper timeout period